src/backends/rendering_context.cpp

```
@@ -463,6 +464,9 @@ bool GLRenderContext::handleGLErrors() const
 	{
 		if(engineData && engineData->getGLError(err))
 		{
+#if defined(ENABLE_GLES2) || defined(ENABLE_GLES3)
+			if(err!=GL_INVALID_ENUM)
+#endif
```

This is an absolute no-go, as it will certainly lead to graphical glitches.
We can't just ignore errors, we have to figure out which OpenGL call is causing the error and fix that call.

---

> This is an absolute no-go, as it will certainly lead to graphical glitches. We can't just ignore errors, we have to figure out which OpenGL call is causing the error and fix that call.

this was temporary as these happened but didn't seem to prevent Super fancy pants from functioning, I haven't been able to figure out a way to trace the errors, I'm fine with removing them but I'll definetely need help to figure out what's happening

---

Well, I normally put a call to handleGLErrors() at several points in the code where OpenGL calls are made to narrow down where the error originates.
Then I set a breakpoint in handleGLErrors() and start the debugger to see which call was the first to cause the error.
Can you describe how to reproduce the error (or does it only occur on Mac) ?

---

From what I remember, just boot fancy pants world 4 and press play on GLES2 or GLES3
It'll fail early enough to say it failed loading a big texture, altho I think that's false

---

It renders fine with GLES3, I have no idea why it happens

---

I fixed it, so now Stage3D rendering works with GLES2, too.
So please rebase and then I will pull this PR